omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptMapController1_9Test.java

```diff
@@ -98,12 +99,4 @@ public void shouldPurgeConceptMap() throws Exception {
 		handle(newDeleteRequest(getURI() + "/" + getUuid(), new Parameter("purge", "")));
 		assertNull(restHelperService.getObjectByUuid(ConceptMap.class, getUuid()));
 	}
-	
-	@Test
-	public void shouldSearchAndReturnAListOfConceptMapTypesMatchingTheQueryString() throws Exception {

```

Comment:
I had to delete this test, because it doesn't make any sense - there was no way to search concept/mapping subresource with get method. This test passes additional arguments and `ConceptMapResource1_9` doesn't implement `Searchable` so it started to fail with adjusted `get` method in `MainSubResourceController`